string words
uint8  sum

